subprojects/android-test/test-screenshot/src/main/java/com/avito/android/test/screenshot_test/internal/BitmapHelpers.kt

```diff
@@ -0,0 +1,60 @@
+package com.avito.android.test.screenshot_test.internal
+
+import android.annotation.SuppressLint
+import android.content.Context
+import android.graphics.Bitmap
+import android.graphics.BitmapFactory
+import android.util.Log
+import java.io.*
```

Comment:
Setup the IDEA to do not use star imports

Reply:
done